import/marc.properties

```diff
@@ -120,3 +120,6 @@ pattern_map.oclc_num.pattern_0 = [(][Oo][Cc][Oo][Ll][Cc][)][^0-9]*[0]*([0-9]+)=>
 pattern_map.oclc_num.pattern_1 = ocm[0]*([0-9]+)[ ]*[0-9]*=>$1
 pattern_map.oclc_num.pattern_2 = ocn[0]*([0-9]+).*=>$1
 pattern_map.oclc_num.pattern_3 = on[0]*([0-9]+).*=>$1
+
+# Work identification keys
+work_keys_str_mv = custom, getWorkKeys(130anp:730anp, 240anpmr:245abn:246abn:247abn, 100ab:110ab:111ac:700ab:710ab:711ac, "([0-9A-Za-zåöäÅÖÄ]+)", "")
```

Comment:
Is the inclusion of `åöäÅÖÄ` a Finnish-specific preference? Any thoughts on whether we should broaden or narrow this for the default distribution?

Reply:
Another question is whether we should have this on by default in the indexing rules, since it does a significant amount of processing and might slightly slow things down. Perhaps it should be moved to marc_local.properties as a commented-out option by default.

Reply:
Perhaps it would be better to list just A-Z by default and give the Finnish/Swedish set as an alternative example.

I'd actually like to propose this to be on by default and enabled in the UI. The reception has been overwhelmingly positive, and even though there are several suggestions on what to add to the keys, they're working very well indeed. But that's just me. With RecordManager I didn't see anything significant regarding performance, but then it's already slower.

Reply:
marc.properties updated, custom example added to marc_local.properties.

Reply:
Thanks! I'm also open to turning this on by default, assuming we don't run into any problems that suggest we shouldn't.